backend/fms_core/template_importer/row_handlers/normalization_planning/normalization_planning.py

```diff
@@ -46,7 +46,8 @@ def process_row_inner(self, source_sample, destination_sample, measurements, rob
                 self.errors['concentration'] = f'A sample or library needs a known concentration to be normalized. QC sample {source_sample_obj.name} first.'
 
             # ensure that the sample source is a library if the norm choice is library
-            if robot['norm_choice'] == LIBRARY_CHOICE and not source_sample_obj.is_library:
+            # If it is a pool we have to check if it is a pool of libraries
```

Comment:
is_library should still cover that case ... ( a logical or between is_library and is_pool_library will always be true if is_library is true and false when is_library is false )